schema/3.0.0/schema.md

```diff
@@ -440,7 +440,9 @@ Curators MUST annotate the following columns in the `obs` dataframe:
     </tr>
     <tr>
       <th>Value</th>
-        <td>categorical with <code>str</code> categories. If unavailable, this MUST be <code>"unknown"</code>.<br><br>This MUST be free-text that identifies a unique individual that data were derived from. It is STRONGLY RECOMMENDED that this identifier be unique to a given individual within the collection of datasets that includes this dataset. 
```

Comment:
In what scenario(s) would we expect 'unknown'?

Reply:
Are there not existing scenarios in the data corpus where there is no donor field or its equivalent?

Reply:
Discussed in slack. Addressed in a7dd8e9616d84f8e8ade1985b49c5514157cc473.